### PR TITLE
feat: add `@cds/core` shim

### PR DESCRIPTION
**Pull request by kevinbuhmann**
This pull request copies the shim from `@cds/core@6.2.5` to `@clr/ui`.

We are moving the shim to the `@clr/ui` package to allow both to be evolved together. Otherwise, changes to the custom property API exposed by `@clr/ui` will require changes in a separate package (and separate repo). This would cause version synchronization difficulties.

How to update:

The file path in applications should be updated as follows:
'@cds/core/styles/shim.clr-ui.min.css' -> '@clr/ui/shim.cds-core.min.css'

That's it, just a simple find and replace.

### CODE REVIEW

File: .storybook/pages/cds-core-shim.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Pages/cds-core shim" />
+
+# @cds/core shim
+
+Clarity provides a theme the enables the `@clr/ui` package to consume the Clarity Core design tokens
+and theme system. Once installed, `@clr/ui` applications will be able to use all Clarity Core
```
Comment (williamernest) [inline .storybook/pages/cds-core-shim.stories.mdx:8]: Should we specify that this shim only applies the colors, and not the entire theme (spacing, typography, etc)?

**Review by williamernest — APPROVED** [review]
One comment on messaging, but we can also change that after the fact without having to cut a new release. Otherwise, LGTM

File: .storybook/pages/cds-core-shim.stories.mdx (hunk 1 above)
Comment (kevinbuhmann) [reply .storybook/pages/cds-core-shim.stories.mdx:8]: Yeah, I will reword this. I don't even think the typography css is even needed to use the shim. It's more a separate thing.

Comment (kevinbuhmann) [reply .storybook/pages/cds-core-shim.stories.mdx]: I have updated this page. https://486--storybook-clarity-design.netlify.app/?path=/story/cds-core-shim--page

Comment (kevinbuhmann) [reply .storybook/pages/cds-core-shim.stories.mdx]: I have updated this page.

- Page: https://486--storybook-clarity-design.netlify.app/?path=/story/cds-core-shim--page
- Changes: https://github.com/vmware-clarity/ng-clarity/compare/b6ff882bee0efcf34692c9a47a1d02f1fe60289b..fc0e47aba5f29b442e0a31695d084974db15e88e

File: .storybook/pages/cds-core-shim.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Import and load the CSS shim file last after `@clr/ui` global CSS. This will ensure that the shim can
+override the `@clr/ui` values appropriately.
+
+```css
+@import 'modern-normalize/modern-normalize.css';
+@import '@cds/core/global.min.css'; // provides the Clarity Core design tokens
+@import '@cds/core/styles/theme.dark.min.css'; // to allow dark mode with cds-theme="dark"
+@import '@clr/ui/clr-ui.min.css'; // @clr/ui CSS (this should already be imported by your application)
+@import '@cds/ui/shim.cds-core.min.css'; // the shim file
```
Comment (kevinbuhmann) [inline .storybook/pages/cds-core-shim.stories.mdx:26]: This path is incorrect. I fixed it in #496.

**Pull request merged**